task9/view.js

```diff
@@ -0,0 +1,97 @@
+class View{
+	viewPost(post, author){
+        let myPost = document.createElement('div');
+        myPost.className = 'post';
+        myPost.innerHTML = "";
+        if(post.hasOwnProperty('photoLink'))
+            myPost.innerHTML += "<img src="+ post.photoLink + " class=\"ava\"> \n";
+        else
+        		myPost.innerHTML += "<img src=https://upload.wikimedia.org/wikipedia/commons/2/2f/No-photo-m.png class=\"ava\"> \n";
+        myPost.innerHTML +=
+            "<span class=\"post_name\">\n" +
+            "<strong>" + post.author + "</strong> \n" +
+            "<div class=\"post_date\">\n" +
+             moment(post.createdAt).format("On MMMM D, YYYY H:m") + 	
+            "\n </div> \n" +
+            "</span> \n";
+        if(author === post.author)
+            myPost.innerHTML +=
+                "<div class=\"post_mybuttons\">\n" +
+                "<button class=\"button\">✎</button>\n" +
+                "<button class=\"button\">␡</button>\n" +
+                "</div>"
+        if(post.hasOwnProperty('hashTags')){
+            let tags = post.hashTags;
+            myPost.innerHTML += "<span class=\"post_hashtags\"> \n";
+            tags.forEach(tag => {
+                myPost.innerHTML += "<span class=\"sep_hashtag\">" + tag + "</span> \n"
+            });
+            myPost.innerHTML += "</span> \n";
+        }
+        myPost.innerHTML +=
+            "<div class=\"post_text\">\n" +
+            post.description +
+            "</div>\n";
+        myPost.innerHTML +=
+            "<div class=\"like\">\n" +
+            "<strong>" + post.likes.length + "</strong>\n" +
+            "<button class=\"button\">❤</button>\n" +
+            "</div>"
+        document.getElementById('fed').append(myPost);
+    }
+        viewFeed(tweets, count, filt = {},author = ""){
+    		document.getElementById('fed').innerHTML = "";
+    		let feed = tweets.getPage(0,count,filt);
+    		for(let i = 0; i < Math.min(count, feed.length); i++){

```

Comment:
I don't think, feed length can be greater than count, since it is one of parameters in `getPage`